pypowsybl/sensitivity.py

```diff
@@ -249,11 +319,62 @@ def set_branch_flow_factor_matrix(self, branches_ids: _List[str], variables_ids:
                 branch_data_frame_index.append(TO_REMOVE)
             else:
                 raise _PyPowsyblError(f'Unsupported factor variable type {type(variable_id)}')
+        return (flatten_variables_ids, branch_data_frame_index)
 
-        _pypowsybl.set_branch_flow_factor_matrix(self._handle, branches_ids, flatten_variables_ids)
```

Comment:
@obrix for backward compatibility, I think we should:
 - keep matrix_id optional so that if not specified we add factors to "default" matrix
 - keep set_branch_flow_factor_matrix that forward to add_.. with default matrix and deprecate it in the API

Reply:
I added the modifications.